packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/app/index.js

```diff
@@ -0,0 +1,6 @@
+const callself = require('callself')
+try {
+  callself.makeTheCall()
+} catch (e) {
+  throw new Error('Referencing exported functions should be possible', { cause: e });
```

Comment:
BTW `cause` is supported starting node 16.9.0 but didn't show up for me in the output. Is Error here not the real Error?